### ISSUE TITLE
Finish failovercache -> DTL rename

### ISSUE THREAD

**Issue by redlicha** [openvstorage/volumedriver#47]
There are still binaries called `failovercache` and `failovercache_helper`.

**Comment by wimpers** [comment]
Assigned to Fargo as you are working on this.

**Comment by wimpers** [comment]
Anything to do here?

**Comment by redlicha** [comment]
TBD: rebase to the latest dev branch and then merging it in / testing it - some changes were done in an automated fashion and especially regressions that affect backward compatibility need to be ruled out.

**Comment by wimpers** [comment]
@redlicha let's go through it an bite the QA bullet.

**Comment by wimpers** [comment]
@redlicha did we already merge this in?

**Comment by redlicha** [comment]
No, it's not merged in yet as it is a change that needs coordination with the framework / other tasks took precendence.
